pokemon_test.go

```diff
@@ -210,3 +210,21 @@ var _ = Describe("Pokemon Data", func() {
 		Expect(pkmn.Data().Name).To(Equal("Arceus"))
 	})
 })
+
+var _ = Describe("RandomGender", func() {
+	DescribeTable("Should always return one gender",
+		func(rate int, gender Gender) {
+			for i := 0; i < 10000; i++ {
+				Expect(RandomGender(uint8(rate))).To(Equal(gender))
+			}
+		},
```

Comment:
This feels like a convoluted way to test this functionality. At this point we're just testing if a RNG is working. We can probably do just fine with 2-3 hard-coded iterations.

Reply:
This is equivalent of hard coding 10000 iterations in a row. This guarantees that `RandomGender` holds the behavior we are expecting.